frog/client/main.html

```diff
@@ -1,5 +1,9 @@
 <head>
   <title>FROG</title>
+  <script src="https://cdn.ravenjs.com/3.17.0/raven.min.js" crossorigin="anonymous"></script>
```

Comment:
Do you know of an easy way to make this configurable? It does not yet have access to Meteor.settings at this stage.